ios/RNCWKWebView.m

```diff
@@ -122,6 +135,8 @@ - (void)didMoveToWindow
     [self addSubview:_webView];
     [self setHideKeyboardAccessoryView: _savedHideKeyboardAccessoryView];
     [self visitSource];
+  } else if (_messagingEnabled) {
```

Comment:
When is this code called?

Should this be NOT _messagingEnabled?

Reply:
It looks like that line was added @ https://github.com/react-native-community/react-native-webview/commit/e0fe48b716e8b9278c6c78e8b0902f7b1ec91eaa

I don't know exactly what that whole conditional block is supposed to do so I left it alone for now. My reading of it (which may be totally off) was something like

```objective-c
if (self.window != nil && _webView == nil) {
  // stuff that happens to setup the webview
} else {
 // stuff that happens to teardown the webview
}
```

If this is accurate, the teardown check `else if (_messagingEnabled)` makes sense, because it's removing the script handler that would have been added in setup if messaging was enabled.